apps/studio/components/layouts/AppLayout/RestrictionBanner.tsx

```diff
@@ -0,0 +1,43 @@
+import { useParams } from 'common'
+import { useRouter } from 'next/router'
+
+import { useOrganizationsQuery } from 'data/organizations/organizations-query'
+import dayjs from 'dayjs'
+import { useSelectedProject } from 'hooks'
+
+/**
+ * Shown on projects in organization which are above their qouta
+ */
+export const RestrictrionBanner = () => {
+  const router = useRouter()
+  const { ref: projectRef } = useParams()
+
+  const project = useSelectedProject()
+
+  const { data } = useOrganizationsQuery()
+
+  const currentOrg = data?.find((org) => org.id === project?.organization_id)
+
+  if (!currentOrg?.restriction_status) {
+    return null
+  }
+  currentOrg
+  return (
+    <div
+      className="flex items-center justify-center gap-x-4 bg-surface-100 py-3 transition text-foreground box-border border-b border-default"
+      style={{ height: '44px' }}
+    >
+      <p className="text-sm">
+        <a href={`/org/${currentOrg.slug}/billing`}>
```

Comment:
I added a link to the banner. But I feel it wouldn't be super clear to the users that the banner is clickable. Let me know if you have a better idea

Reply:
pushed an update to the UI for this